library/query_label_context_injection.yml

```diff
@@ -0,0 +1,28 @@
+id: RP-5
+
+info:
+  name: Label Context Injection
+  severity: high
+  description: Label Injection is caused by using label variables in inline scripts
+  full-description: |
+    Creating a new pull request could be submitted with a label that contains special characters.
+    By using a label variable in an inline script, an attacker can inject arbitrary code into the build process.
+  references:
+    - https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions
+  tags:
+    - injection
+    - unauthenticated
+
+query: |
+  MATCH (w:Workflow)-[*]->(d:StepCodeDependency)
+    WHERE (
+        "issues" in w.trigger OR
+        "issue_comment" in w.trigger OR
+        "pull_request_target" in w.trigger
```

Comment:
pull_request trigger too?

Reply:
with `pull_request`, even when you managed to inject, you won't have any permissions to do anything